raiden_libs/types.py

```diff
@@ -1 +1,8 @@
 # -*- coding: utf-8 -*-
+from typing import NewType
+
+T_Address = bytes
```

Comment:
If it is bytes, then it'll break a thing or two for mypy.

And just for the record, I can't say I am a big fan of using _any_ bin-encoded data, unless absolutely necessary. I just don't see any advantage here. web3 expects hex-encoded string; any log print would require conversion bin->str; bytes are hard to read in debugger; until there are hard metrics performance gain should be considered negligible.

I am really curious If there's any real reason behind this choice.

Reply:
Good point. Forgot to change this. I'm not sure where the preference for `bytes` is coming from, but I'd also vote for using string addresses.